src/config.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Gregorio configuration headers. Copyright (C) 2015 Elie Roux
```

Comment:
Just a detail: I tend to change the copyright to "The Gregorio project". Maybe we should discuss it with a broader audience, but I tend to prefer it: legally speaking it's less precise, but it's mostly for the symbol: I don't want Gregorio to be my personal software, with my name, etc. I'd prefer being just one of the authors, and to let other people participate without feeling underneath me...